easybuild/easyconfigs/s/Snippy/Snippy-4.6.0-GCC-9.3.0-Java-13-Python-3.8.2.eb

```
@@ -0,0 +1,60 @@
+# Author: Jasper Grimm (UoY)
+
+easyblock = 'CmdCp'
+
+name = 'Snippy'
+version = '4.6.0'
+local_pyversuffix = '-Python-%(pyver)s'
+local_javaversuffix = '-Java-%(javaver)s'
+versionsuffix = local_javaversuffix + local_pyversuffix
+
+homepage = 'https://github.com/tseemann/snippy'
+description = """Snippy finds SNPs between a haploid reference genome and your NGS sequence
+ reads. It will find both substitutions (snps) and insertions/deletions (indels).
+ Rapid haploid variant calling and core genome alignment."""
+
+toolchain = {'name': 'GCC', 'version': '9.3.0'}
+
+github_account = 'tseemann'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['7264e3819e249387effd3eba170ff49404b1cf7347dfa25944866f5aeb6b11c3']
+
+dependencies = [
+    ('Java', '13', '', True),
+    ('Python', '3.8.2'),
+    ('Perl', '5.30.2'),
+    ('BioPerl', '1.7.7'),
+    ('any2fasta', '0.4.2'),
+    ('BWA', '0.7.17'),
+    ('BCFtools', '1.10.2'),
+    ('BEDTools', '2.29.2'),
+    ('freebayes', '1.3.5', local_pyversuffix),
```

```suggestion
    ('freebayes', '1.3.5', local_pyversuffix),
    ('gzip', '1.10'),
```

I have `Found gzip - /usr/bin/gzip` in the log so we should as `gzip` as a dependency.